### ISSUE TITLE
something's broken

### ISSUE THREAD

**Issue by hb0nd** [StrandedKitty/streets-gl#173]
![image](https://github.com/StrandedKitty/streets-gl/assets/26939824/87d269d3-e933-4f4a-a029-770a45b8db9e)

https://streets.gl/#51.47299,-0.02143,7.50,5.00,3997.83

**Comment by StrandedKitty** [comment]
Interesting, I don't seem to reproduce this on my device. Maybe it's environment-specific.

**Comment by s3n-w6i** [comment]
![image](https://github.com/StrandedKitty/streets-gl/assets/37022952/dae2b3cb-87f7-42e8-93cc-86f46b2789e5)
Can reproduce. But for me, there are different floating buildings. They are, however, consistent across force reloads (with cache cleared). It works as expected in Brave, but not Firefox.

Firefox 120.0
nixos
Intel -- Mesa Intel(R) Graphics (RPL-P)
4.6 (Compatibility Profile) Mesa 23.0.3

**Comment by svenpilz** [comment]
![image](https://github.com/StrandedKitty/streets-gl/assets/4508103/e9dd0621-e107-4d9e-afcc-000400106a0c)

I cannot reproduce this with Firefox.

Firefox 120.0
arch
Intel -- Mesa Intel(R) UHD Graphics (TGL GT1)
4.6 (Core Profile) Mesa 23.2.1-arch1.2

**Comment by hb0nd** [comment]
hardware which i can reproduce on:

Firefox 120 (flatpak)
Fedora
AMD -- Radeon™ 760M Graphics
4.6 (Core Profile) Mesa 23.2.1

very weird

**Comment by kayD** [comment]
I can reproduce this, e.g. here: https://streets.gl/#49.75082,10.11855,45.00,0.00,1669.57

![Screenshot_20231228_213415](https://github.com/StrandedKitty/streets-gl/assets/316554/b84647f9-02bb-4e2b-a8ba-49de9c614330)

Only happens on Firefox 120.0 on my Ubuntu system, Vivaldi 6.5 works fine!

Side note, I do not know if it is important: you can see a small set of trees on the top left corner of this screenshot. It is floating (can only be seen if moving around a bit), in about 50m height. It is one of a series of about 6 floating objects in the area, all surprisingly near the two high buildings.

Edit: I just noticed that in the original screenshot from @RedAuburn (initial post from Nov 24: https://github.com/StrandedKitty/streets-gl/issues/173#issue-2009711782 ), near the center of the image, there also seem to be two patches of foliage floating in the air, next to the building.

**Comment by kayD** [comment]
I also could - at first - not reproduce the issue at https://streets.gl/#51.47299,-0.02143,7.50,5.00,3997.83 (looking at @svenpilz) - all buildings looked OK.
When I panned around, some (few) huge buildings appeared.

So I closed FF 120.0 (each time a new private window), and reopened, and now saw a few huge buildings. I.e., I can reproduce it now.
Then: pressing F5 to reload -> same buildings are huge
Then: closing/reopening FF -> other buildings are huge
